pkg/realmd/operation.js

```diff
@@ -488,6 +484,21 @@ define([
         };
 
         return element;
+    }
+
+    module.button = function button() {
```

Comment:
Removing it breaks any other packages that might have been using the button. I doubt anything was, but it seemed fairly innocuous to leave there for the off chance that someone needs it. But since we are making no stability guarantees I went ahead and removed it. It's easy enough to add it later should someone complain.